fixbackend/auth/user_repository.py

```diff
@@ -60,6 +60,23 @@ async def get_by_email(self, email: str, *, session: Optional[AsyncSession] = No
             user = await db.get_by_email(email)
             return user.to_model() if user else None
 
+    async def search(self, query: str) -> Sequence[User]:
```

Comment:
the name `query` is a bit misleading - `email_part`?

Reply:
could be a uuid as well, so query is more generic